neural/neural_discern_run_script.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# % load_ext autoreload
+# % autoreload 2
+
+import torch.multiprocessing as mp
+
+import argparse
+import os
+import datetime
+import pandas as pd
+import torch
+from pytorch_pretrained_bert import BertModel, BertForPreTraining, BertConfig
+
+from neural.data_processor import DataDictProcessor
+from neural.dataset import generate_docpartition_per_question, validate_q_docpartitions, compute_class_weights_per_fold_
+from neural.model import BertEmbedder, generate_sents_embeds_from_docs
+from neural.run_workflow import generate_models_config, HyperparamConfig, hyperparam_model_search_parallel, \
+    get_best_config_from_hyperparamsearch, train_val_run, train_val_run_one_question, test_run
+from neural.utilities import ReaderWriter, create_directory
+
+
+# might have to be immediately after import torch.multiprocessing as mp
+mp.set_start_method("spawn", force=True)
+
+
+def read_pickles(directory):
+    # processed data directory
+    data_dir = create_directory('proc_data_uncased', directory)
```

Comment:
I think I found the issue regarding BERT and BioBERT. It seems you are always reading from `proc_data_uncased` which is the `uncased` tokenization of the dataset. While BioBERT need the `cased` version. I checked down the `main` logic and it seems it always reads from `uncased` folder (unless I am missing something).

Reply:
oooooh! good catch!